molgenis-web/pom.xml

```diff
@@ -10,6 +10,7 @@
     <packaging>jar</packaging>
 
     <name>web</name>
+    <description>Web framework without plugins. Despaghettified molgenis-core-ui.</description>
```

Comment:
Personally I'd remove the second park, describe what the module contains rather than where it came from.